appengine/appengine.bzl

```diff
@@ -238,30 +238,30 @@ APPENGINE_BUILD_FILE = """
 # BUILD file to use the Java AppEngine SDK with a remote repository.
 java_import(
     name = "jars",
-    jars = glob(["**/*.jar"]),
+    jars = glob(["appengine-java-sdk-1.9.38/lib/**/*.jar"]),
```

Comment:
Here and below: can we use a * instead of the version number to avoid chaniging it as each version, or at least factor out the version number.